site2/docs/getting-started-standalone.md

```diff
@@ -1,278 +1,150 @@
-
-:::tip
-
-If you want to use built-in connectors and tiered storage offloaders, you can install them according to the following instructions：
-* [Install built-in connectors (optional)](#install-built-in-connectors-optional)
-* [Install tiered storage offloaders (optional)](#install-tiered-storage-offloaders-optional)
-Otherwise, skip this step and perform the next step [Start Pulsar standalone](#start-pulsar-standalone). Pulsar can be successfully installed without installing built-in connectors and tiered storage offloaders.
-
-:::
-
-### Install built-in connectors (optional)
-
-Since `2.1.0-incubating` release, Pulsar releases a separate binary distribution, containing all the `built-in` connectors.
-To enable those `built-in` connectors, you can download the connectors tarball release in one of the following ways:
-
-* download from the Apache mirror <a href="pulsar:connector_release_url" download>Pulsar IO Connectors @pulsar:version@ release</a>
-
-* download from the Pulsar [downloads page](pulsar:download_page_url)
-
-* download from the Pulsar [releases page](https://github.com/apache/pulsar/releases/latest)
-
-* use [wget](https://www.gnu.org/software/wget):
-
-  ```shell
-  wget pulsar:connector_release_url/{connector}-@pulsar:version@.nar
-  ```
-
-After you download the NAR file, copy the file to the `connectors` directory in the pulsar directory. 
-For example, if you download the `pulsar-io-aerospike-@pulsar:version@.nar` connector file, enter the following commands:
+By default, the standalone mode runs a RocksDB instance for metadat storage. If you'd prefer to start a cluster with standalone ZooKeeper server, set `PULSAR_STANDALONE_USE_ZOOKEEPER` to 1:
 
 ```bash
-mkdir connectors
-mv pulsar-io-aerospike-@pulsar:version@.nar connectors
-
-ls connectors
-pulsar-io-aerospike-@pulsar:version@.nar
-...
+PULSAR_STANDALONE_USE_ZOOKEEPER=1 bin/pulsar standalone
 ```
 
-:::note
+These directories are created once you started the Pulsar cluster.
 
-* If you are running Pulsar in a bare metal cluster, make sure `connectors` tarball is unzipped in every pulsar directory of the broker (or in every pulsar directory of function-worker if you are running a separate worker cluster for Pulsar Functions).
-* If you are [running Pulsar in Docker](getting-started-docker.md) or deploying Pulsar using a docker image (e.g. [K8S](deploy-kubernetes.md) or [DC/OS](https://dcos.io/), you can use the `apachepulsar/pulsar-all` image instead of the `apachepulsar/pulsar` image. `apachepulsar/pulsar-all` image has already bundled [all built-in connectors](io-overview.md#working-with-connectors).
+* `data` directory stores all data created by BookKeeper and RocksDB.
+* `logs` directory contains all server-side logs.
 
-:::
+## Create a topic
 
-### Install tiered storage offloaders (optional)
+Pulsar stores messages in topics. It's good practice to explicitly create them before using them, even if Pulsar can automagically create them when referenced.
```

Comment:
```suggestion
Pulsar stores messages in topics. It's good practice to explicitly create them before using them, even if Pulsar can automatically create them when referenced.
```
Suggest clarifying the first `them`. Otherwise, users may confuse whether it's talking about messages or topics.

Reply:
Updated at https://github.com/apache/pulsar/pull/17475/commits/79aad25a91777831c7ed0d63375fe633d51c1525